windows/scripts/extract_kernels.py

```diff
@@ -26,7 +26,9 @@
 ------------
 
 Requires 7-Zip (http://www.7-zip.org/) and that the ``7z`` executable is
-available on the ``PATH``.
+available on the ``PATH``. Note that to extract WIM images from Windows 8 and
+newer, you must use 7-Zip version 7.30 or newer (see
+https://sourceforge.net/p/sevenzip/discussion/45798/thread/248aac94/).
```

Comment:
Could you check the version in the script? I prefer when the error message indicates what's wrong.